src/Files.App/Helpers/ShellContextMenuHelper.cs

```diff
@@ -173,15 +173,19 @@ async void InvokeShellMenuItem(ContextMenu contextMenu, object? tag)
 					case "install" when isFont:
 						{
 							var userFontDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "Microsoft", "Windows", "Fonts");
-							var destName = Path.Combine(userFontDir, Path.GetFileName(contextMenu.ItemsPath[0]));
-							Win32API.RunPowershellCommand($"-command \"Copy-Item '{contextMenu.ItemsPath[0]}' '{userFontDir}'; New-ItemProperty -Name '{Path.GetFileNameWithoutExtension(contextMenu.ItemsPath[0])}' -Path 'HKCU:\\Software\\Microsoft\\Windows NT\\CurrentVersion\\Fonts' -PropertyType string -Value '{destName}'\"", false);
+							foreach (string path in contextMenu.ItemsPath)
+							{
+								var destName = Path.Combine(userFontDir, Path.GetFileName(path));
+								Win32API.RunPowershellCommand($"-command \"Copy-Item '{path}' '{userFontDir}'; New-ItemProperty -Name '{Path.GetFileNameWithoutExtension(path)}' -Path 'HKCU:\\Software\\Microsoft\\Windows NT\\CurrentVersion\\Fonts' -PropertyType string -Value '{destName}'\"", false);
```

Comment:
Can this be merged into a common method? You can pass a parameter to set if it applies to all users.